### ISSUE TITLE
Channel Usage above 200% : Issue or feature ?

### ISSUE THREAD

**Issue by romarysonrier** [br101/horst#111]
Hello,

When snifing  MCS15  traffic ( bandwidth test between my Android phone and 2.4GHz Wifi AP) I get 230% of Channel Usage , is this an error in airtime channel usage computation ? or a feature due to MIMO  or multiple spatial stream ? 

In other words, what could cause airtime channel usage to be above 100% ?

Capture device chipset is Alfa AWUS036ACH (rtl8812au with https://github.com/aircrack-ng/rtl8812au driver version v5.6.4.2).

Regards,
Romary

**Comment by br101** [comment]
Yes, it's a bug in the airtime calculation. No time to fix it, sorry.
